packages/node_modules/@webex/webex-core/test/integration/spec/services/services.js

```diff
@@ -476,6 +484,23 @@ describe('webex-core', () => {
 
         assert.notEqual(priorityUrl, nextPriorityUrl);
       });
+
+      it('should reset hosts once all hosts have been marked failed', () => {
```

Comment:
This test seems to validate the `_getPriorityHostUrl` feature works. Would a simpler test be to loop through `.hosts` and set failed, then call `markFailedUrl` and check the failed status of all the hosts again?

Reply:
Ok, I see you're doing this in the unit tests.

Reply:
In that case, should I leave this as is?